### PR TITLE
Fix matchExpression Readability

### PR DESCRIPTION
**Pull request by dustman9000**
Update matchExpressions to be more readable and fix non-FR expressions.

### CODE REVIEW

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 1]
```diff
@@ -339,12 +357,33 @@ objects:
...
+          operator: NotIn
+          values: ${{ALERTS_SKIP_LEGALENTITY_IDS}}
+        - key: hive.openshift.io/cluster-region
+          operator: NotIn
+          values:
+            - "cn-north-1"
+            - "cn-northwest-1"
+        - key: api.openshift.com/fedramp
+          operator: NotIn
+          value: "true"
```
Comment (gsleeman) [inline hack/olm-registry/olm-artifacts-template.yaml:386]: should be a list (i think)

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 2]
```diff
@@ -275,15 +275,33 @@ objects:
...
-        - { key: hive.openshift.io/cluster-region, operator: NotIn, values: ["cn-north-1", "cn-northwest-1"] }
-        - { key: api.openshift.com/fedramp, operator: DoesNotExist }
+        - key: hive.openshift.io/cluster-region
+          operator: NotIn
+          values:
+            - "cn-north-1"
+            - "cn-northwest-1"
+        - key: api.openshift.com/fedramp
+          operator: NotIn
+          value: "true"
```
Comment (gsleeman) [inline hack/olm-registry/olm-artifacts-template.yaml:304]: should be a list (i think)

File: hack/olm-registry/olm-artifacts-template.yaml (hunk 1 above)
Comment (dustman9000) [reply hack/olm-registry/olm-artifacts-template.yaml]: fixed

**Pull request merged**